src/Components/InputTodo.js

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { runToast } from './toast/Toast';
+
+const InputTodo = ({ list, setList }) => {
+    const [form, setForm] = useState('');
+    const handleChange = (e) => {
+        setForm(e.target.value);
+    };
+    const handleInput = () => {
+        const createItem = () => {
+            setList([
+                ...list,
+                {
+                    id: uuidv4(),
+                    content: form,
+                    done: false,
+                },
+            ]);
+        };
+
+        form // 입력한 값이 있고,
+            ? !list.find((item) => item.content === form) //중복되는 값이 없을 때
+                ? createItem()
+                : runToast('동일한 내용으론 작성할 수 없어요!')
+            : runToast('할 일을 입력해주세요!');
+
+        //입력창 초기화
+        setForm('');
+    };
+
+    const handleEnter = () => {
+        if (window.event.keyCode == 13) {
+            handleInput();
+        }
+    };
```

Comment:
```suggestion
    const handleEnter = (event) => {
        if (event.keyCode == 13) {
            handleInput();
        }
    };
```
이벤트 핸들러니까 이벤트 객체를 argument로 받아 사용하는 것이 더 좋을 것 같습니다.